### PR TITLE
Add app config to admin portal make app config configurable during run time. 

### PR DESCRIPTION
**Pull request by thivi**

### CODE REVIEW

File: apps/admin-portal/src/configs/routes.ts [hunk 1]
```diff
@@ -142,6 +149,7 @@ const APP_ROUTES: RouteInterface[] = [
     {
         component: AuthLayout,
         icon: null,
+        id: "login",
```
Comment (jeradrutnam) [inline apps/admin-portal/src/configs/routes.ts]: Duplicate `id`, is it correct?

File: apps/admin-portal/src/configs/routes.ts [hunk 2]
```diff
@@ -150,6 +158,7 @@ const APP_ROUTES: RouteInterface[] = [
     {
         component: AuthLayout,
         icon: null,
+        id: "logout",
```
Comment (jeradrutnam) [inline apps/admin-portal/src/configs/routes.ts]: Duplicate `id`, is it correct?

File: apps/admin-portal/src/helpers/app-config-context.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (jeradrutnam) [inline apps/admin-portal/src/helpers/app-config-context.ts]: Remove new line here

File: modules/core/src/models/route.ts [hunk 4]
```diff
@@ -34,9 +34,15 @@ export interface RouteInterface {
      * Icon to be displayed on the side panel.
      */
     icon?: any;
+
+    /**
+     * A key to uniquely identify the route with
+     */
+    id: string;
```
Comment (DimalChandrasiri) [inline modules/core/src/models/route.ts]: Add a new line here.

**Pull request merged**